RFS/src/test/java/com/rfs/integration/SnapshotStateTest.java

```diff
@@ -60,6 +61,7 @@ public void tearDown() throws Exception {
     @Test
     public void SingleSnapshot_SingleDocument() throws Exception {
         // Setup
+        final var testContext = TestContext.noOtelTracking();
```

Comment:
Nit; this should be refactored into the SimpleRestoreFromSnapshot_ES_7_10.  I'll do this later - thanks for merging this in.